parsons/zoom/zoom.py

```diff
@@ -151,6 +151,8 @@ def __process_poll_results(self, tbl: Table) -> Table:
         `Returns`:
             Parsons Table
         """
+        if tbl.num_rows == 0:
```

Comment:
Totally pedantic and not important, but since the merge of #830, could use the more python `if not len(tbl):`